src/ripple/app/main/Application.cpp

```diff
@@ -1607,6 +1608,19 @@ bool ApplicationImp::setup()
         }
     }
 
+    if (shardStore_)
+    {
+        using namespace boost::filesystem;
+
+        auto stateDb(
+            RPC::ShardArchiveHandler::getStateDirectory(*config_)
+            / "stateDatabase");
+
+        if(exists(stateDb) && is_regular_file(stateDb) &&
+           !RPC::ShardArchiveHandler::hasInstance())
+            RPC::ShardArchiveHandler::getInstance(*this);
```

Comment:
Consider this format:
```        if (exists(stateDb) &&
            is_regular_file(stateDb) &&
            !RPC::ShardArchiveHandler::hasInstance())
        {
            RPC::ShardArchiveHandler::getInstance(*this);
        }
```

Reply:
Consider catching exceptions, logging and returning false;

Reply:
Done